app/initialize.js

```diff
@@ -1,20 +1,20 @@
 import application from './application';
-import { syncFiles } from './libs/file';
-import scdl from './libs/soundcloud';
+import Polyglot from 'node-polyglot';
 
+function initLocale() {
+    let locale = document.documentElement.getAttribute('lang');
```

Comment:
Yup, 'cauz your `lang` attribute on HTML tag in your _index.html_ file as `en` as value. What if your cozy is setted to display FR locale?